components/Steps/Summary.jsx

```diff
@@ -18,12 +18,13 @@ const SummarySection = ({ formData, name, title, slug }) => (
     <SummaryList
       list={Object.entries(formData[name]).map(([key, value]) => ({
         title: getInputProps(name, key).label,
-        value:
-          typeof value === 'object'
-            ? Object.values(value).map(MultiValue)
-            : typeof value === 'boolean'
-            ? JSON.stringify(value)
-            : value
+        value: Array.isArray(value)
+          ? value.map(v => MultiValue(v.split('/').pop()))
+          : typeof value === 'object'
+          ? Object.values(value).map(MultiValue)
+          : typeof value === 'boolean'
+          ? JSON.stringify(value)
+          : value
```

Comment:
These nested turnaries are a bit hard to read.

Reply:
Do you think a `switch` would be more clear?

Reply:
Merging this, if you think it's relevant I can create another PR :)